types.go

```diff
@@ -577,25 +577,25 @@ type TxStatus struct {
 }
 
 type TokenDetail struct {
-	Name                       string             `json:"name"`
-	MaxSupply                  string             `json:"maxSupply"`
-	TotalSupply                string             `json:"totalSupply"`
-	CirculatingSupply          string             `json:"circulatingSupply"`
-	SzDecimals                 int                `json:"szDecimals"`
-	WeiDecimals                int                `json:"weiDecimals"`
-	MidPx                      string             `json:"midPx"`
-	MarkPx                     string             `json:"markPx"`
-	PrevDayPx                  string             `json:"prevDayPx"`
-	Genesis                    TokenDetailGenesis `json:"genesis"`
-	Deployer                   string             `json:"deployer"`
-	DeployGas                  string             `json:"deployGas"`
-	DeployTime                 string             `json:"deployTime"`
-	SeededUsdc                 string             `json:"seededUsdc"`
-	NonCirculatingUserBalances [][]string         `json:"nonCirculatingUserBalances"`
-	FutureEmissions            string             `json:"futureEmissions"`
+	Name                       string              `json:"name"`
+	MaxSupply                  string              `json:"maxSupply"`
+	TotalSupply                string              `json:"totalSupply"`
+	CirculatingSupply          string              `json:"circulatingSupply"`
+	SzDecimals                 int                 `json:"szDecimals"`
+	WeiDecimals                int                 `json:"weiDecimals"`
+	MidPx                      string              `json:"midPx"`
+	MarkPx                     string              `json:"markPx"`
+	PrevDayPx                  string              `json:"prevDayPx"`
+	Genesis                    *TokenDetailGenesis `json:"genesis"`
+	Deployer                   *string             `json:"deployer"`
+	DeployGas                  *string             `json:"deployGas"`
+	DeployTime                 *string             `json:"deployTime"`
+	SeededUsdc                 string              `json:"seededUsdc"`
+	NonCirculatingUserBalances [][]string          `json:"nonCirculatingUserBalances"`
+	FutureEmissions            string              `json:"futureEmissions"`
 }
 
 type TokenDetailGenesis struct {
-	UserBalances          [][]string `json:"userBalances"`
-	ExistingTokenBalances [][]string `json:"existingTokenBalances"`
+	UserBalances          [][]string      `json:"userBalances"`
+	ExistingTokenBalances [][]interface{} `json:"existingTokenBalances"`
```

Comment:
why `interface{}`? If data types are diverse for this field, considering  using the  MixedValue or MixedArray available here